src/components/PrivacyModal/PrivacyModal.tsx

```diff
@@ -0,0 +1,114 @@
+import { Modal, Typography, Box, Button, Link } from '@common/components'
+import { FormGroup, FormControlLabel, Checkbox } from '@mui/material'
+import { usePrivacyModal as _usePrivacyModal, PrivacyModalHookReturn } from './PrivacyModal.hooks'
+import { useTranslation } from 'react-i18next'
+import React from 'react'
+//zukünftig in die einstellung eine möglichkeit zum cookie declinen einbauen
+//was passiert wenn accept geklickt wird?
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 400,
+  bgcolor: 'background.paper',
+  border: '2px solid #000',
+  boxShadow: 24,
+  p: 4,
+  display: 'flex'
+}
+
+/**
+ * @typedef {Object} PrivacyModalProps
+ * @property {function} usePrivacyModal - Hook used for the Modal logic upon clicking a button.
+ */
+
+export type PrivacyModalProps = {
+  usePrivacyModal?: () => PrivacyModalHookReturn
+}
+
+/**
+ * PrivacyModal shows a Modal for the user to read and accept the PrivacyPolicy and other.
+ * The PrivacyModal gets shown upon entering haski. Gets rendered in App.
+ * @returns {JSX.Element} - The Modal
+ * @category Components
+ */
+
+const PrivacyModal = ({ usePrivacyModal = _usePrivacyModal }: PrivacyModalProps) => {
+  const { t } = useTranslation()
+  const [open, setOpen] = React.useState(true)
+  const [checked, setCheck] = React.useState(false)
+  const { onAcceptHandler } = usePrivacyModal()
+
+  //Disable backdropClick so the Modal only closes via the buttons
+  const handleClose = (event: React.MouseEvent<HTMLElement>, reason: string) => {
```

Comment:
Nicht verwendete Events als _ darstellen. Siee Komponentenguide

```suggestion
  const handleClose = (_: React.MouseEvent<HTMLElement>, reason: string) => {
```